spec/features/admin_authenticates_spec.rb

```diff
@@ -59,6 +75,11 @@ def expect_to_not_have_sign_out_link_for_student
   expect(page).to have_no_xpath("//a[@href='/sign_out']")
 end
 
+def navigate_to_admin_login(admin)
```

Comment:
Unused method argument - `admin`. If it's necessary, use `_` or `_admin` as an argument name to indicate that it won't be used. You can also write as `navigate_to_admin_login(*)` if you want the method to accept any arguments but don't care about them.